starknet_py/tests/e2e/client/fixtures/transactions.py

```diff
@@ -111,7 +111,7 @@ async def block_with_declare_v2_number(
     return declare_v2_receipt.block_number
 
 
-@pytest_asyncio.fixture(scope="package")
+@pytest_asyncio.fixture(scope="function")
```

Comment:
The `get_new_state_update` test using this fixture was malfunctioning. Since the fixture uses another fixture, `account`, the test was invoked 4 times:
1. [gateway_client-gateway_account] 
2. [full_node_client-gateway_account] 
3. [full_node_client-full_node_account] 
4. [gateway_client-full_node_account] 

The first and third test replaced the class successfully, but since the scope of the fixture was marked as `"package"`, the second and fourth test failed, because the class has already been replaced and last assert in the test failed:
```
assert (
        ReplacedClass(contract_address=contract_address, class_hash=class_hash)
        in state_update.state_diff.replaced_classes
    )
```

Reply:
Great 👍 there's some [problems with this exact test on the target branch](https://github.com/software-mansion/starknet.py/actions/runs/5146247441/jobs/9265329165?pr=1025#step:10:488) - let's hope this fixes them